### PR TITLE
PLAT-824: refactor method and tests

### PR DESCRIPTION
**Pull request by MariaVasilenko**
refactor CreatePipe() and benchmark tests
tests is failed

### CODE REVIEW

File: ledger-core/network/nds/msgdelivery/uniproto_utils_test.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+	var dispatcher uniserver.Dispatcher
+	controller.RegisterWith(dispatcher.RegisterProtocol)
+
+	vf := TestVerifierFactory{}
+
+	srv := uniserver.NewUnifiedServer(&dispatcher, TestLogAdapter{t})
+	srv.SetConfig(config)
+	srv.SetIdentityClassifier(idWithPortFn)
+
+	// This is min value for NodeID, o is not allowed
```
Comment (AndreyBronin) [inline ledger-core/network/nds/msgdelivery/uniproto_utils_test.go:58]: "0" not "o" is not allowed, instead

File: ledger-core/network/nds/msgdelivery/uniproto_utils_test.go [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+func getServerByIndex(idx int) *UnitProtoServer {
+	if idx < 1 || idx > len(servers) {
+		panic("")
+	}
+
+	return servers[idx-1]
+}
+
+type UnitProtoServer struct {
```
Comment (AndreyBronin) [inline ledger-core/network/nds/msgdelivery/uniproto_utils_test.go:33]: UniprotoServer - from word "unified" not from "unit"

File: ledger-core/network/nds/msgdelivery/uniproto_utils_test.go (hunk 1 above)
Comment (MariaVasilenko) [reply ledger-core/network/nds/msgdelivery/uniproto_utils_test.go]: fixed

File: ledger-core/network/nds/msgdelivery/uniproto_utils_test.go (hunk 2 above)
Comment (MariaVasilenko) [reply ledger-core/network/nds/msgdelivery/uniproto_utils_test.go]: fixed

**Review by AndreyBronin — APPROVED** [review]

**Pull request closed without merging**